### DISCUSSION TITLE
retrieve holdout set

### DISCUSSION THREAD

**Discussion by rdataforge** [pycaret/pycaret#2565] [Q&A] [answered]
reading tutorial here,
https://github.com/pycaret/pycaret/blob/master/tutorials/Regression%20Tutorial%20Level%20Beginner%20-%20REG101.ipynb
 seems that Pycaret saves apart a holdout set to validate at the end of the process.
I would like to know if there is any way of retrieving such holdout set for further inspection and how to be sure is not used anywhere.

thanks

**Comment by moezali1** [comment, marked as answer]
@rdataforge `get_config('X_test')` will give you access to test/hold-out set.

Hope this helps.

**Reply by rdataforge** [reply]
Thanks @moezali1 
I assumed this holdout set is never used. right?

**Reply by moezali1** [reply]
Yes correct. Not used in training at all.

**Reply by rdataforge** [reply]
in finalize_model perhaps?

**Reply by rdataforge** [reply]
something is wrong.   get_config('X_test') same shape as test set. I assume is the same data.
 How come is not used at least for CV?
